### PR TITLE
Rename sender to clan

### PR DESCRIPTION
**Pull request by XrioBtw**
Sender, now clan, holds the clan name, not the message sender name

### CODE REVIEW

**Comment by deathbeam** [comment]
Sender is better name. The thing that it is used only in CC is different, but still, naming it clan is not correct imo. For name associated to message, there is `name`, in same logic then name should be renamed so sender, what is not correct as well.

**Comment by XrioBtw** [comment]
Sender is not a descriptive name for the clan name and is confusing. E.g. the result of sender from a message in the RuneLite cc will be Rl.

**Comment by SomeoneWithAnInternetConnection** [comment]
I just took a peek at the client, and it turns out that the field that is currently named `sender` only contains a string if the message is in clanchat (in which case it's the clan name). Otherwise it seems to be null.

**Comment by deathbeam** [comment]
Yeah I am aware of that (that is what I wrote before) but i quite like sender a bit more still. But if you want to change it, change it everywhere, not only in chat message (that means addChatMessage, sendGameMessage, QueueMessage)

**Comment by XrioBtw** [comment]
I don't think my replacement was successful. I will let Adam decide if this should be replaced or not.

File: deobfuscator/src/test/resources/net/runelite/runeloader/injection_130.json [hunk 1]
```diff
@@ -923,7 +923,7 @@
 		}, {
 			"className": "s",
 			"getterMethodDesc": "()Ljava/lang/String;",
-			"getterName": "getSender",
+			"getterName": "getClan",
```
Comment (Adam-) [inline deobfuscator/src/test/resources/net/runelite/runeloader/injection_130.json]: definitely don't rename this

**Comment by deathbeam** [comment]
runescape-client was removed, closing due to age.

**Pull request closed without merging**